apps/arena/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Arena.MixProject do
   def project do
     [
       app: :arena,
-      version: "0.1.0",
+      version: "0.1.14",
```

Comment:
Arbitrary number? Jajaja

Reply:
Yeah, but we need to sync front and back when this pr gets merged